src/components/button/style/index.scss

```diff
@@ -0,0 +1,10 @@
+@import "mixin";
+@import '../../../styles/index.scss';
```

Comment:
I imported this here to make it works for the variable bellow. But probably need to find a better way to add global `index.scss` style file to Storybook so that we won't have to import this in every component stylesheet.